pyproject.toml

```diff
@@ -19,19 +19,27 @@ structlog = "^21.5.0"
 colorama = "^0.4.4"
 mypy-boto3-s3 = "^1.21.23"
 owid-catalog = "^0.2.6"
-
+flake8-type-annotations = "^0.1.0"
```

Comment:
Shouldn't flake8-type-annotations be in `dev-dependencies`? Same for bumpversion?

Reply:
Ups, yes - totally right!